src/Microsoft.ML.Core/Utilities/Utils.cs

```diff
@@ -534,7 +534,15 @@ public static int[] GetRandomPermutation(Random rand, int size)
             return res;
         }
 
-        public static bool AreEqual(Single[] arr1, Single[] arr2)
+        public static void Shuffle<T>(Random rand, Span<T> rgv)
```

Comment:
Sorting functions -- this was in the middle of a batch of `AreEqual` implementation. I moved it up near the rest of the randomization utils.